bayesian/utility/user_utils.py

```diff
@@ -0,0 +1,66 @@
+"""Definition of all utility and db interactions for user management."""
+import datetime
+import logging
+
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy import insert, update
+
+from f8a_worker.models import (UserDetails)
+from bayesian import rdb
+
+logger = logging.getLogger(__file__)
+
+
+def get_user(uuid_val):
+    """Get User."""
+    try:
+        query = rdb.session.query(UserDetails).filter(UserDetails.user_id == uuid_val)
+        user = query.one()
+        return user
+    except SQLAlchemyError:
+        rdb.session.rollback()
+        logger.exception(f"Error fetching user with id {uuid_val}")
+        raise UserException("Error fetching user")
+
+
+def create_user(uuid_val, user_source):
+    """Create User."""
+    try:
+        insert_user_stmt = insert(UserDetails).values(user_id=uuid_val, user_source=user_source,
```

Comment:
I think you should add strftime to add the date in proper format that we want.

Reply:
The dates seem to be in the right format, i checked using pgweb. Maybe it is doing some conversion, i will check this.

Reply:
with strftime, you will be able to store it in a particular format. Then later when we want to query using these values, it will be very easy for us.

Reply:
This converts it to a string but the data type in the DB is datetime.  If we convert to string we will loose the benefits that the DB provides with datetime.

Reply:
oh ok. schema is datetime. then we cannot use strftime.